spec/generators/dashboard_generator_spec.rb

```diff
@@ -33,8 +33,8 @@ class Foo < ApplicationRecord
           load file("app/dashboards/foo_dashboard.rb")
           attrs = FooDashboard::ATTRIBUTE_TYPES
 
-          expect(attrs[:id]).to eq(Administrate::Field::Number)
           expect(attrs[:created_at]).to eq(Administrate::Field::DateTime)
+          expect(attrs[:id]).to eq(Administrate::Field::Number)
```

Comment:
Why this change?

Reply:
I'm not sure what I originally made this change. I've undone it and the tests are still passing.